docs/init.rst

```diff
@@ -193,12 +194,12 @@ Since the validators runs *after* the instance is initialized, you can refer to
 This example also shows of some syntactic sugar for using the `attr.validators.and_` validator: if you pass a list, all validators have to pass.
 
 ``attrs`` won't intercept your changes to those attributes but you can always call `attr.validate` on any instance to verify that it's still valid:
+When using `define` or `frozen`, ``attrs`` will run the validators even when setting the attribute.
```

Comment:
```suggestion
When using `define` or :func:`~attr.frozen`, ``attrs`` will run the validators even when setting the attribute.
```

Reply:
I see, great. Single backticks has a special meaning where it attempts to actually create a link?

Reply:
Single backticks together with our default_role means basically "guess, mofo"

See: #571

Reply:
(I could've used `attr.frozen`, but then I wouldn't be able to use `~`)